src/mainwindow.py

```diff
@@ -53,15 +68,16 @@ def start(self):
         if not self.session:
             self.session = aiohttp.ClientSession()
 
+        cachesize = int(self.config['cachesize'])
         # Run 10 times to get 10 cats
-        for i in range(10):
+        for i in range(cachesize):
             # initialize a dict of cat data
             cat_data = dict()
 
             # randomly make jumpscares happen, but not on the first image
-            if randint(1, 10) == 5 and i:
+            if randint(1, cachesize) == 5 and i:
```

Comment:
This is meant to be hardcoded in so that it's a 1/10 chance for a jump scare. This will change based on the cache size which we don't want.

Reply:
Got it, I'll fix these.